projects/bp-graphql/src/operations/getExhibition.ts

```diff
@@ -0,0 +1,119 @@
+import { Operation, graphql } from '../Operation.js';
+
+export default {
+  document: graphql`
+    query getExhibition($exhibitionId: ID!) {
+      exhibition(id: $exhibitionId) {
+        data {
+          id
+          attributes {
+            title
+            introduction
+            epilog
+            is_published
+            title_picture {
+              data {
+                id
+                attributes {
+                  subtitle
+                  picture {
+                    data {
+                      id
+                      attributes {
+                        media {
+                          data {
+                            id
+                            attributes {
+                              width
+                              height
+                              formats
+                              url
+                              updatedAt
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            idealot_pictures {
+              data {
+                id
+                attributes {
+                  subtitle
+                  picture {
+                    data {
+                      id
+                      attributes {
+                        media {
+                          data {
+                            id
+                            attributes {
+                              width
+                              height
+                              formats
+                              url
+                              updatedAt
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            exhibition_sections(sort: "order:asc") {
+              data {
+                id
+                attributes {
+                  title
+                  text
+                  order
+                  exhibition_pictures(sort: "order:asc") {
+                    data {
+                      id
+                      attributes {
+                        order
+                        subtitle
+                        picture {
+                          data {
+                            id
+                            attributes {
+                              media {
+                                data {
+                                  id
+                                  attributes {
+                                    width
+                                    height
+                                    formats
+                                    url
+                                    updatedAt
```

Comment:
We need the provider attribute to check if the picture is either uploaded locally or on aws, else pictures from aws wont load
```suggestion
                            attributes {
                              width
                              height
                              formats
                              url
                              updatedAt
                              provider
                            }
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
            idealot_pictures {
              data {
                id
                attributes {
                  subtitle
                  picture {
                    data {
                      id
                      attributes {
                        media {
                          data {
                            id
                            attributes {
                              width
                              height
                              formats
                              url
                              updatedAt
                              provider
                            }
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
            exhibition_sections(sort: "order:asc") {
              data {
                id
                attributes {
                  title
                  text
                  order
                  exhibition_pictures(sort: "order:asc") {
                    data {
                      id
                      attributes {
                        order
                        subtitle
                        picture {
                          data {
                            id
                            attributes {
                              media {
                                data {
                                  id
                                  attributes {
                                    width
                                    height
                                    formats
                                    url
                                    updatedAt
                                    provider
```